client/features/example.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  As any user,
+  when I go to homepage,
+  I see a resource.
+
+  Scenario: I see the Book resource
```

Comment:
I'm not sure that we should encourage this as it's a BDD good practice. We should either:

* write real BDD scenario (functional ones, not technical ones)
* or write technical E2E tests, with Jest instead of cucumber

I would prefer using cucumber with proper scenarios written in natural language, it would be a good example for the community.

Reply:
Requires #63